### ISSUE TITLE
run speed?

### ISSUE THREAD

**Issue by zyg11** [Media-Smart/vedadet#52]
Hello! I found the speed was slow almost near 1.4 second per image When I  tried the eval code . Is it normal? My environment is v100.

**Comment by MinliangLin** [comment]
@zyg11 I think it is not normal behavior. I test some face detection tasks on GeForce RTX 2070 and the speed is about 0.2s per image.

**Comment by vokhidovhusan** [comment]
Yes, it looks very slow. It is taking 1.2 second for per image with RTX 2080 when I run test.py file.
